samples/skills/QASkill/MemoryQuery/skprompt.txt

```diff
@@ -0,0 +1,7 @@
+Someone wants to know the following about a GitHub repository:
+{{$input}}
+
+Please answer the above question given the following info:
+{{textmemoryskill.recall $input}}
```

Comment:
the order of the two blocks seems wrong and the prompt is missing the command to ignore general knowledge embedded in the model. 
Usually LLMs perform better providing context first, and a targeted question after.

Did you try this prompt?
```
{{textmemoryskill.recall $input}}
---
Considering only the information above, which has been loaded from a GitHub repository, answer the following.
Question: {{$input}}

Answer:
```

Reply:
Switch to your prompt